public/app/features/explore/LogRow.tsx

```diff
@@ -44,6 +58,28 @@ const FieldHighlight = onClick => props => {
   );
 };
 
+const getLogRowStyles = (theme: GrafanaTheme, state: State) => {
```

Comment:
This will be called for each row, and we render up to 1000. I'm less worried about the call, but more about a class being created for the normal row state that only has `position: relative` in it. Perhaps that can be avoided?

Reply:
The call is optimised now